lib/index.js

```diff
@@ -18,6 +18,7 @@ const memoize = require("./util/memoize");
 /** @typedef {import("../declarations/WebpackOptions").Externals} Externals */
 /** @typedef {import("../declarations/WebpackOptions").FileCacheOptions} FileCacheOptions */
 /** @typedef {import("../declarations/WebpackOptions").LibraryOptions} LibraryOptions */
+/** @typedef {import("../declarations/WebpackOptions").MemoryCacheOptions} MemoryCacheOptions */
```

Comment:
Why only memory? Maybe we should export and filesystem cache?

Reply:
`FileCacheOptions` is already exported 2 lines above.

Reply:
My bad, sorry :smile: